modules/oadp-installing-dpa-1-2-and-earlier.adoc

```diff
@@ -102,6 +106,7 @@ spec:
 <9> Specify the name of the `Secret` object that you created. If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the backup location.
 <10> Specify a snapshot location, unless you use CSI snapshots or Restic to back up PVs.
 <11> The snapshot location must be in the same region as the PVs.
+<12> Specify the name of the `Secret` object that you created. If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the snapshot location. If your backup and snapshot locations use different credentials, you create separate profiles in the `credentials-velero` file.
```

Comment:
```suggestion
<12> Specify the name of the `Secret` object that you created. If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the snapshot location. If your backup and snapshot locations use different credentials, create separate profiles in the `credentials-velero` file.
```
Consider changing this sentence to use the imperative to parallel the imperative in the first sentence.